bin/newman.js

```diff
@@ -34,6 +34,8 @@ program
     .option('--env-var <value>',
         'Allows the specification of environment variables via the command line, in a key=value format',
         util.cast.memoizeKeyVal, [])
+    .option('-s,--skip-iteration <value>',
```

Comment:
Please do not reserve a single character parameter for a tertiary feature. The single characters should be kept free for major top level features.

Also, could you check the pattern of parameters for iterations and use something along those lines for the double word parameter?